great_expectations/experimental/datasources/pandas_datasource.py

```diff
@@ -201,19 +234,97 @@ def add_csv_asset(
         data_path: PathStr,
         regex: Union[str, re.Pattern],
         order_by: Optional[BatchSortersDefinition] = None,
-    ) -> CSVAsset:
```

Comment:
@Kilo59 I would prefer that this method would be called `add_csv_data_asset()` -- spelled out for clarity.  Thanks.

Reply:
I don't want to change anything about the public API in this PR.